lib/logflare/erl_sys_mon.ex

```diff
@@ -48,20 +49,22 @@ defmodule Logflare.ErlSysMon do
   end
 
   def handle_info({:monitor, pid, _type, _meta} = msg, state) when is_pid(pid) do
-    pid_info =
-      pid
-      |> Process.info(:dictionary)
-      |> case do
-        {:dictionary, dict} when is_list(dict) ->
-          Keyword.take(dict, [:"$ancestors", :"$initial_call"])
-
-        other ->
-          other
-      end
-
-    Logger.warning(
-      "#{__MODULE__} message: " <> inspect(msg) <> "|\n process info: #{inspect(pid_info)}"
-    )
+    Tasks.async(fn ->
```

Comment:
Doesn't seem right to me to create a task just to log something

And if we're really getting this many messages then really not sure about creating tasks for each one